keyboards/planck/keymaps/hvp/config.h

```diff
@@ -0,0 +1,55 @@
+#define TAPPING_TERM 150
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+
+#ifndef CONFIG_USER_H
+#    define CONFIG_USER_H
+
+#    include "config_common.h"
+
+/* USB Device descriptor parameter */
+#    define VENDOR_ID 0xFEED
+#    define PRODUCT_ID 0x6060
+#    define MANUFACTURER OLKB
+#    define PRODUCT Planck
+#    define DESCRIPTION A compact ortholinear keyboard
```

Comment:
This entire block should be removed.   

And you should add a `#pragma once` to the top of the file.

Reply:
Updated